DoLahTest/ParserUnitTest.cpp

```diff
@@ -53,6 +56,14 @@ namespace DoLahTest {
         }
     }
 
+    std::string UNHANDLED_COMMAND_MESSAGE = "Command not handled";
```

Comment:
You may want to begin this section with a comment like "Unit Test for individual Parser components". Same for the above section, "Unit Test for main Parser class".